mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
```

Comment:
The function name in the controller is `exports.removeOrder` but it's being called as `remove` in the route. It should be `removeOrder` instead of `remove`.